inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/service/DisplayMessageJobIntentService.kt

```diff
@@ -63,6 +71,8 @@ internal class DisplayMessageJobIntentService : JobIntentService() {
      * This method displays message on UI thread.
      */
     private fun displayMessage(message: Message, hostActivity: Activity) {
+        if (verifyContexts(message) == false) return
```

Comment:
[MINOR] why not use `!verifyContexts(message)`?

Reply:
Just so it's more clear. Maybe the callback could have a different name or maybe make a naw local var to hold output of the closure